scripts/build_and_load_runner.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
I LOVE THIS IT WILL MAKE LIFE EASIER THANK YOU

Reply:
HAPPY TO HELP YEAH I WASNT ENJOYING DOING THIS MANUALLY EITHER